docs/howto/configure/update-env.md

```diff
@@ -13,7 +13,7 @@ have to manually build & push it after making a change.
 
 2. Make the changes you need to make to the environment, and git commit it.
 
-3. Run `python3 deploy.py build`. This will build the image and push it to
+3. Run `python3 deployer build`. This will build the image and push it to
```

Comment:
I think the build command needs to be passed the cluster-name too.

Reply:
That is the recent merge #358, right?

Reply:
yes

Reply:
Good catch, fixed.